src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -195,85 +199,88 @@ public void clickVisibilityOptionForResponseCommentAndSave(String idString, int
         commentEditForm.findElement(By.className("col-sm-offset-5")).findElement(By.tagName("a")).click();
         ThreadHelper.waitFor(1000);
     }
-    
-    public boolean verifyAllResultsPanelBodyVisibility(boolean visible){
+
+    public boolean verifyAllResultsPanelBodyVisibility(boolean visible) {
         int numOfQns = browser.driver.findElements(By.cssSelector(".panel-heading+.panel-collapse")).size();
         assertTrue(numOfQns > 0);
-        
+
         // Wait for the total duration according to the number of collapse/expand intervals between questions
         ThreadHelper.waitFor((numOfQns * 50) + 1000);
-        
-        for(WebElement e : browser.driver.findElements(By.cssSelector(".panel-heading+.panel-collapse"))){
-            if(e.isDisplayed() != visible){
+
+        for (WebElement e : browser.driver.findElements(By.cssSelector(".panel-heading+.panel-collapse"))) {
+            if (e.isDisplayed() != visible) {
                 return false;
             }
         }
         return true;
     }
-    
-    public boolean verifyAllStatsVisibility(){
-        for(WebElement e : browser.driver.findElements(By.className("resultStatistics"))){
-            if(e.getCssValue("display").equals("none")){
+
+    public boolean verifyAllStatsVisibility() {
+        for (WebElement e : browser.driver.findElements(By.className("resultStatistics"))) {
+            if (e.getCssValue("display").equals("none")) {
                 return false;
             }
         }
         return true;
     }
-    
+
     public void deleteFeedbackResponseComment(String commentIdSuffix) {
         WebElement commentRow = browser.driver.findElement(By.id("responseCommentRow" + commentIdSuffix));
-        commentRow.findElement(By.tagName("form"))
-            .findElement(By.tagName("a")).click();
+        commentRow.findElement(By.tagName("form")).findElement(By.tagName("a")).click();
         ThreadHelper.waitFor(1500);
     }
-    
-    public void verifyCommentRowContent(String commentRowIdSuffix, String commentText, String giverName) {
+
+    public void verifyCommentRowContent(String commentRowIdSuffix,
+                                        String commentText, String giverName) {
         WebDriverWait wait = new WebDriverWait(browser.driver, 30);
         WebElement commentRow;
-        try{
-            commentRow = wait.until(ExpectedConditions.presenceOfElementLocated(By.id("responseCommentRow" + commentRowIdSuffix)));
-        } catch (TimeoutException e){
+        try {
+            commentRow = wait.until(ExpectedConditions.presenceOfElementLocated(By.id("responseCommentRow"
+                                                                                      + commentRowIdSuffix)));
```

Comment:
This is OK too. (The break is at a higher-level than before although the indentation doesn't follow any rule; when you break a deeply-nested element, it becomes harder to read).
```
commentRow = wait.until(ExpectedConditions.presenceOfElementLocated(
                                                 By.id("responseCommentRow"+ commentRowIdSuffix)));
```